### PR TITLE
Page Object Model

### PR DESCRIPTION
**Pull request by didiermg**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,6 +32,7 @@
     "jasmine-spec-reporter": "^4.2.1",
     "protractor": "^5.4.2",
     "save-dev": "^2.0.0",
+    "selenium-webdriver": "^4.0.0-alpha.5",
```
Comment (sorregom) [inline package.json]: why do you need this?

File: test/buy-tshirt.spec.ts [hunk 2]
```diff
@@ -1,41 +1,48 @@
-import { $, browser } from 'protractor';
+import { browser } from 'protractor';
+import { MenuContentPage, SignInPage, AddressStepPage, ShippingStepPage, PaymentStepPage,
```
Comment (sorregom) [inline test/buy-tshirt.spec.ts]: order each import by line to easy read

**Review by sorregom — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (didiermg) [reply package.json]: It was added when I updated the webdriver. It has been removed.

File: test/buy-tshirt.spec.ts (hunk 2 above)
Comment (didiermg) [reply test/buy-tshirt.spec.ts]: I have ordered according to suggestion.

**Review by sorregom — APPROVED** [review]
OK

**Pull request merged**